### PR TITLE
ARM-CI : Fix ARM Emulator vm runs out of space

### PR DESCRIPTION
**Pull request by sjsinju**
Now, ARM Emulator failed by out of space on tests (6676)
To resolve,a temp folder in the job's workspace would be bound to /tmp. and then 'runtest.sh' run by some directories bound from /tmp.

### CODE REVIEW

**Comment by sjsinju** [comment]
@dotnet-bot test Linux ARM Emulator Cross Debug Build please

**Comment by jasonwilliams200OK** [comment]
@jkotas, @jashook, apparently it has fixed the active issue #6676. which is causing all the recent PRs to fail ARM Emulator build.  Can this be merged with squash? I guess further changes to this can be incorporated separately.

**Comment by jkotas** [comment]
@jasonwilliams200OK agree

File: tests/scripts/arm32_ci_script.sh [hunk 1]
```diff
@@ -180,6 +180,10 @@ function mount_emulator {
     mount_with_checking "-o bind" "/dev/pts" "$__ARMRootfsMountPath/dev/pts"
     mount_with_checking "-t tmpfs" "shm"     "$__ARMRootfsMountPath/run/shm"
     mount_with_checking "-o bind" "/sys"     "$__ARMRootfsMountPath/sys"
+        if [ ! -d "$__ARMRootfsMountPath/bindings/tmp" ]; then
```
Comment (jkotas) [inline tests/scripts/arm32_ci_script.sh]: @sjsinju Could you please fix the indentation when you get a chance?

**Comment by jashook** [comment]
This PR has been reverted by @mmitche with #6923, thank you for that. @sjsinju, unfortunately this was merged too soon. Your bound directory /tmp is incorrect, it will still continue to use the small main drive of the Azure VM. This is the same problem we had before, just by chance did not repo this time. Can you please bind to /mnt to use the larger temporary drive. Preferably a temp folder created and then cleaned up under /mnt so that we do not overwrite accidentally the workspace created by jenkins automation at that location.

@mmitche please feel free to add any other suggestions.

/cc @RussKeldorph @russellhadley @jasonwilliams200OK

**Comment by jashook** [comment]
Also note that #6676 should be kept open until we can confirm that we are cleaning up the arm emulator sufficiently and the newly used bound temp directory.

**Pull request merged**